pkg/ddc/alluxio/ufs_internal.go

```diff
@@ -97,6 +97,152 @@ func (e *AlluxioEngine) shouldMountUFS() (should bool, err error) {
 	return should, err
 }
 
+func (e *AlluxioEngine) getMounts() (result_dataset []string, result_mountted []string, err error) {
```

Comment:
`result_dataset`   

about code style suggest  to follow the same standard
[CodeReviewComments#initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Reply:
Thanks, have refined it.